### PR TITLE
New Registries - Last batch

### LINKED ISSUE #109: Add Registry - Audio Config
https://www.isdcf.com/site/dcnc-appendix-4-audio-configuration/

### PR DESCRIPTION
**Pull request by SteveLLamb**
New registries closes #109, #110, #111, #112, #113 

- Audio Configs
- Projector Aspect Ratios
- KDM Forensic Flags
- CPL Metadata Extensions
- ULs

### CODE REVIEW

File: src/main/schemas/audioconfigs.schema.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              "type": "string"
+          },
+          "MCATagSymbol": {
+              "type": "string"
+          },
+          "extension": {
+            "$ref": "#/definitions/extension"
+          },
+          "definingDoc": {
+              "type": "array"
```
Comment (palemieux) [inline src/main/schemas/audioconfigs.schema.json:24]:
```suggestion
              "type": "array",
              "items": {
                "type": "number"
              }
```

File: src/main/schemas/projectoraspectratios.schema.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  "definitions": {
+    "dcncCode" : {
+        "type": "string",
+        "pattern": "^\\[?[A-Z0-9]\\]?"
+    },
+    "cplMetadata": {
+      "type": "object",
+      "properties": {
+        "definingDoc": {
+          "type": "array"
```
Comment (palemieux) [inline src/main/schemas/projectoraspectratios.schema.json:15]:
```suggestion
              "type": "array",
              "items": {
                "type": "number"
              }
```

File: src/main/schemas/projectoraspectratios.schema.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          "$ref": "#/definitions/4K"
+        }
+      },
+      "required": [ "definingDoc", "2K", "4K"]
+    },
+    "2K": {
+      "type": "object",
+      "properties": {
+        "ScreenAspectRatio": {
+          "type": "string"
```
Comment (palemieux) [inline src/main/schemas/projectoraspectratios.schema.json:30]:
```suggestion
          "type": "string",
          "pattern": "^\\d+ \\d+?"
```

File: src/main/schemas/projectoraspectratios.schema.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+          "$ref": "#/definitions/MainPictureStoredArea"
+        }
+      },
+      "required": [ "ScreenAspectRatio", "MainPictureStoredArea"]
+    },
+    "4K": {
+      "type": "object",
+      "properties": {
+        "ScreenAspectRatio": {
+          "type": "string"
```
Comment (palemieux) [inline src/main/schemas/projectoraspectratios.schema.json:42]:
```suggestion
          "type": "string",
          "pattern": "^\\d+ \\d+?"
```

File: src/main/schemas/uls.schema.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      "definingDoc": {
+        "type": "string"
+      },
+      "contact": {
+        "type": "string"
+      },
+      "usage": {
+        "type": "string"
+      },
+      "deprecated": {
```
Comment (palemieux) [inline src/main/schemas/uls.schema.json:29]:
```suggestion
      "obsolete": {
```

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/schemas/audioconfigs.schema.json (hunk 1 above)
Comment (SteveLLamb) [reply src/main/schemas/audioconfigs.schema.json:24]: These values aren't only numbers, the contain values such as:
- "SMPTE ST 428-12"
- "https://www.d-box.com/wp-content/uploads/2020/09/124-915-0005.pdf"

File: src/main/schemas/projectoraspectratios.schema.json (hunk 2 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json:15]: These values aren't only numbers, the contain values such as:
- "SMPTE ST 428-12"

File: src/main/schemas/projectoraspectratios.schema.json (hunk 3 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json:30]: Oh, good one! I also added in an item on the test for this as well.

File: src/main/schemas/projectoraspectratios.schema.json (hunk 4 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json:42]: Same as above, added I also added in an item on the test for this as well.

File: src/main/schemas/projectoraspectratios.schema.json (hunk 3 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json]: Addressed in f5a6107

File: src/main/schemas/projectoraspectratios.schema.json (hunk 4 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json]: Addressed in f5a6107

File: src/main/schemas/uls.schema.json (hunk 5 above)
Comment (SteveLLamb) [reply src/main/schemas/uls.schema.json]: Done, changed in f5a6107

File: src/main/schemas/projectoraspectratios.schema.json (hunk 2 above)
Comment (palemieux) [reply src/main/schemas/projectoraspectratios.schema.json:15]: I meant `"type": "string"`

Apologies.

File: src/main/schemas/audioconfigs.schema.json (hunk 1 above)
Comment (palemieux) [reply src/main/schemas/audioconfigs.schema.json:24]: I meant `"type": "string"`

Apologies.

Comment (SteveLLamb) [reply src/main/schemas/audioconfigs.schema.json]: All good, updated in 23dea0a

File: src/main/schemas/projectoraspectratios.schema.json (hunk 2 above)
Comment (SteveLLamb) [reply src/main/schemas/projectoraspectratios.schema.json]: All good, updated in 23dea0a

**Review by palemieux — APPROVED** [review]

**Comment by SteveLLamb** [comment]
closes #110, closes #111, closes #112, closes #113

**Pull request merged**